src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -30,6 +30,10 @@
 namespace chip {
 namespace Messaging {
 
+#ifndef MIN
+#define MIN(a, b) (((a) < (b)) ? (a) : (b))
+#endif
```

Comment:
nit: there's a generic chip::min() in CodeUtils.h (not sure why we don't use std::min(), though).

Reply:
acked